10-inference.Rmd

```diff
@@ -0,0 +1,806 @@
+```
+
+Notice the column `replicate` is indicating the replicate with which each
+Timbit belongs. Since we took 1000 samples of size 40, there are 1000 replicates.
+
+```{r 11-example-proportions6, echo = TRUE, message = FALSE, warning = FALSE}
+sample_estimates <- samples %>% 
+    group_by(replicate) %>% 
+    summarise(sample_proportion = sum(flavour == "chocolate") / 40)
+head(sample_estimates)
+tail(sample_estimates)
+```
+
+```{r 11-example-proportions7, echo = TRUE, message = FALSE, warning = FALSE,fig.cap = "Sampling distribution of the sample proportion for sample size 40"}
+sampling_distribution <-  ggplot(sample_estimates, aes(x = sample_proportion)) +
+    geom_histogram(fill="#0072B2", color="#e9ecef", binwidth = 0.05) +
+    xlab("Sample proportions") 
+sampling_distribution
+```
+
+The sampling distribution appears to be bell-shaped with one peak. It is
+centered around `r round(mean(sample_estimates$sample_proportion),1)` and the
+sample proportions range from about `r round(min(sample_estimates$sample_proportion), 1)` to 
+about `r round(max(sample_estimates$sample_proportion), 1)`. In fact, we can calculate
+the mean and standard deviation of the sample proportions. 
+
+```{r 11-example-proportions8, echo = TRUE, message = FALSE, warning = FALSE}
+sample_estimates %>% 
+  summarise(mean = mean(sample_proportion), sd = sd(sample_proportion))
+```
+
+We notice that the sample proportions are centred around the population
+proportion value. The standard deviation of the sample proportions 
+is `r round(sd(sample_estimates$sample_proportion), 3)`.
+
+<!--- > **Note:** If random samples of size $n$ are taken from a population, $\hat{p}$ will be approximately Normal with mean $p$ and standard deviation $\sqrt{\frac{p(1-p)}{n}}$ as long as the sample size $n$ is large enough such that $np$ and $n(1 - p)$ are at least 10, where $p$ is the population proportion, $\hat{p}$ is the sample proportion and $n$ is the sample size. 
+--->
+
+### Sampling distributions for means 
+
+In the previous section, our variable of interest&mdash;Timbit flavour&mdash;was
+*categorical*, and the population parameter of interest was the proportion of chocolate
+Timbits. What if we wanted to infer something about a population of *quantitative* variables instead?
+As mentioned in the introduction to this chapter, there are many choices of population parameter
+for each type of observed variable. In this section, we will study the case where we are interested
+in the population mean of a quantitative variable.
+
+In particular, we will look at an example using data from Airbnb, an online
+marketplace for arranging or offering places to stay. The dataset contains
+Airbnb listings for Vancouver, Canada, in September 2020 
+from [Inside Airbnb](http://insideairbnb.com/). 
+Let's imagine (for learning purposes) that our dataset represents the population of all Airbnb rental listings in Vancouver,
+and we are interested in the population mean price per night.
```

Comment:
This comes too late, let's move our question (well let's actually write an inferential question for this case) up to the third sentence of the first paragraph of this section.

Reply:
moved up to first paragraph, added question we want to answer